packages/common-all/src/constants/configs/publishing.ts

```diff
@@ -279,4 +279,8 @@ export const PUBLISHING: DendronConfigEntryCollection<DendronPublishingConfig> =
       desc: "Define custom sidebar",
       value: undefined,
     },
+    searchDefaultBehavior: {
```

Comment:
nit: maybe `defaultSearchBehavior`? or simply `searchMode`? (since lot of our configs are there to provide configurable defaults but don't have `default` in their name)